docs/docs/operate/profiling.md

```diff
@@ -0,0 +1,48 @@
+Mantis allows you to directly connect profiling tools to workers in order to do performance or memory profiling. This
+can be helpful for debugging OOMs. Below are the steps to start profiling your running Mantis job:
+
+ 1. Download a profiling tool. We recommend [Zulu Mission Control (ZMC)](https://www.azul.com/products/zulu-mission-control/).
+ 1. Untar ZMC
+    ```
+    tar -xzvf <zulu_mission_control_package>.tar.gz
+    ```
+ 1. Run zmc.
+    ```
+    note
+    On Mac the `Zulu Mission Control` app is actually a directory even though it has an icon.
+    The `zmc` executable is inside and needs to be called directly.
+    ```
+    ```
+    <your dir>/<zulu_mission_control_package>/Zulu\ Mission\ Control.app/Contents/MacOS/zmc
+    ```
+ 1. Find a worker you want to debug in your job and click on it to get its **ip address** and **debug port** In this
+    example these are `100.82.159.148` and `7156` respectively.
+
+    <img src="../images/profiling/workerDebugURL.png" alt="Worker Debug URL" width="600"/>
+
+ 1. Setup an SSH tunnel with local portforwarding for your worker and port. The tunnel command for the example worker
```

Comment:
should the tunnel command be `ssh -L 7156:127.0.0.1:7156 100.82.159.148` for accessing the port locally? Also, may be a note that this is only needed if the worker ip cannot be hit directly.

Reply:
This. I just do this normally. https://gist.github.com/jeffchao/ac392c016aff17977c890f4c3101c7f2

Reply:
Added note about direct access. Switched to using 127.0.0.1 . I think that's the destination ip. So if the worker's debug port is listening on 127.0.0.1 or it's host ip, then both should work as long as your sshing into the same host. Tested with our setup and both worked.